app/javascript/components/textual_summary/table_list_view.jsx

```diff
@@ -3,40 +3,25 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import MiqStructuredList from '../miq-structured-list';
 
-const simpleRow = (row, i, colOrder) => (
-  <tr key={i} className="no-hover">
-    {colOrder.map((col, j) => <td key={j}>{row[col]}</td>)}
-  </tr>
-);
-
-const clickableRow = (row, i, colOrder, rowLabel, onClick) => (
-  <tr key={i}>
-    {colOrder.map((col, j) => (
-      <td key={j} title={rowLabel}>
-        <a href={row.link} onClick={(e) => onClick(row, e)}>
-          {`${row[col]}`}
-        </a>
-      </td>
-    ))}
-  </tr>
-);
-
-const renderRow = (row, i, colOrder, rowLabel, onClick) => (
-  row.link ? clickableRow(row, i, colOrder, rowLabel, onClick) : simpleRow(row, i, colOrder)
-);
-
```

Comment:
I think we'd be fine to backport this since these methods should have been removed over 2 years ago, but it has nothing to do with the fix so I'll break it up into two prs and then we can decide after.